### ISSUE TITLE
Blur my Shell does not save my changes

### ISSUE THREAD

**Issue by Paco-grc** [aunetx/blur-my-shell#616]
Hello,

I have a problem with Blur my Shell. I lowered the blur intensity of the default pipeline so that it applies to the Preview ("Aperçu"). This renders as I want but the extension cannot keep my setting.

I created my own pipeline but it's the same. I set the Radius to "30" and it always ends up going back to its default value of 60 at some point.

How do I make the extension keep my settings indefinitely.

Best regards,
Paco

**Comment by Babib3l3l** [comment]
i get the same issue, but only when i lock my pc and turn it back on, on startup it works fine otherwise

**Comment by Paco-grc** [comment]
Indeed, I just tested and the problem surely comes from there.

I modify the pipeline parameters and they are correctly taken into account.
However, as soon as I lock my session, the default pipeline settings apply.

**Comment by Babib3l3l** [comment]
yep, i dont have the exact same issue as you, my problem is that the blur on the app menu doesnt save and gets reset to default, but it comes from the same bug.
